lib/v2/screens/profile_screens/guardians/guardians_tabs/interactor/viewmodels/guardians_events.dart

```diff
@@ -77,3 +77,21 @@ class OnRemoveGuardianTapped extends GuardiansEvent {
   @override
   String toString() => 'OnRemoveGuardianTapped : { guardian: $guardian }';
 }
+
+class InitOnboardingGuardian extends GuardiansEvent {
+
+  @override
+  String toString() => 'ShowOnboardingGuardian';
```

Comment:
to string does not match class name

Reply:
fix, I changed the event name and forgot to change the to string.